pkg/assets/config/test-keeper.yaml

```diff
@@ -134,3 +135,4 @@ skip_validation_for:
 
   # Our own configurations
   - 'regex{{test-keeper\.y[a]?ml$}}'
```

Comment:
is this line still necessary when the configs should be inside of the directory that is excluded?